src/drivers/github.go

```diff
@@ -43,7 +43,7 @@ func (d *githubCodeHostingDriver) GetNewPullRequestURL(branch string, parentBran
 	if parentBranch != git.GetMainBranch() {
 		toCompare = parentBranch + "..." + branch
 	}
-	return fmt.Sprintf("%s/compare/%s?expand=1", d.GetRepositoryURL(), toCompare)
+	return fmt.Sprintf("%s/compare/%s", d.GetRepositoryURL(), toCompare)
```

Comment:
I think `?expand=1` is important as it allows you to immediately fill in the PR title and description which otherwise would first require clicking on "Create Pull Request"

Reply:
Okay. The other drivers also use `?`. I am looking for a way to make this work in the Windows shell.